busybox/content.md

```diff
@@ -32,8 +32,8 @@ This `Dockerfile` will allow you to create a minimal image for your statically c
 
 This image contains BusyBox built against various "libc" variants (for a comparison of "libc" variants, [Eta Labs has a very nice chart](http://www.etalabs.net/compare_libcs.html) which lists many similarities and differences).
 
--	`busybox:glibc`: [glibc from Debian](https://packages.debian.org/jessie/libc6) (which is then included in the image)
--	`busybox:musl`: [musl from Alpine](https://pkgs.alpinelinux.org/package/main/x86_64/musl) (statically compiled)
--	`busybox:uclibc`: [uClibc](https://uclibc.org) via [Buildroot](https://buildroot.org) (statically compiled)
+-	`%%IMAGE%%:glibc`: [glibc from Debian](https://packages.debian.org/jessie/libc6) (which is then included in the image)
+-	`%%IMAGE%%:musl`: [musl from Alpine](https://pkgs.alpinelinux.org/package/main/x86_64/musl) (statically compiled)
```

Comment:
I noticed the `x86_64` and then found that the link fails. :cry:  Maybe `https://pkgs.alpinelinux.org/packages?name=musl` instead?